modules/creating-infra-machines.adoc

```diff
@@ -104,9 +117,25 @@ spec:
         path: /etc/infratest
 ----
 
-.  Apply the machine config, then verify that the infra-labeled nodes are updated with the configuration file:
+..  Apply the machine config to the infra-labeled nodes:
 +
 [source,terminal]
 ----
 $ oc create -f infra.mc.yaml
 ----
+
+. Confirm that your new machine config pool is available.
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
+[source,terminal]
+----
+NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT
+infra    rendered-infra-b446e3cf9a1cc9be2c775fb742c41154    True      False      False      0
```

Comment:
Complete ignoramus question: Is 0 typically the correct output for MACHINECOUNT at this point?

Reply:
@yuqi-zhang Can you help with this question?  ^^

Reply:
When you first create the MCP, the machine count is 0. Eventually, it comes up as 1.

Reply:
It somewhat depends on your ordering of operations, application of machineconfigs don't affect this, only node labelling.

If you created the MCP first before scaling up infra nodes/moving workers to infra, it should be 0. If you labelled a node first and then create the MCP, it should almost immediately +1 infra -1 worker.